eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -55,98 +64,70 @@ public RegionWSServer() {
 //    @ApiOperation(httpMethod = "GET", value = "Retrieves all the variants from region", response = QueryResponse.class)
     public QueryResponse getVariantsByRegion(@PathVariable("regionId") String regionId,
                                              @RequestParam(name = "species") String species,
-                                             @RequestParam(name = "studies", required = false) String studies,
+                                             @RequestParam(name = "studies", required = false) List<String> studies,
```

Comment:
As far as I know Spring is able to split a string in comma-separated values so nothing should break, would be more an extension.

And as @tomdcsmith said, the other endpoint supports it and this is the only way to provide a consistent API for the Variant Browser.